src/buyingcatalogue/templates/configmap.yaml

```diff
@@ -41,4 +41,5 @@ data:
   document-container: {{ $container | quote }}
   run-id: {{ $runId | quote }}
   mp-supplier-url: {{ printf "%s/marketing/supplier/solution/" $pbUrl }}
-  mp-authority-url: {{ printf "%s/marketing/authority/solution/" $pbUrl }}
\ No newline at end of file
+  mp-authority-url: {{ printf "%s/marketing/authority/solution/" $pbUrl }}
+  of-ac-url: {{ printf "%s/order" $pbUrl }}
```

Comment:
Should this be "%s/order" or "%s/order/" - not sure if it's significant, but to be consistent

Reply:
I think it should have the trailing `/`, I'll update